index.bs

```diff
@@ -4777,6 +4777,92 @@ which should make use of the existing browser permissions framework for the Geol
 
 The privacy principles in [[!FIDO-Privacy-Principles]] also apply to this specification.
 
+
+## De-anonymization prevention measures ## {#sctn-privacy-attacks}
+
+[INFORMATIVE]
+
+Many aspects of the design of the [=Web Authentication API=] are motivated by privacy concerns. The main concern considered in
+this specification is the protection of the user's personal identity, i.e., the identification of a physical person or a
+correlation of separate identities as belonging to the same physical person. Although the [=Web Authentication API=] does not use
+or provide and form of global identity, the following kinds of potentially correlatable identities are used:
+
+- The user's [=credential IDs=] and [=credential public keys=].
+
+    These are registered by the [=[RP]=] and subsequently used by the user to prove possession of the corresponding [=credential
+    private key=], and thus ownership of the [=public key credential=] identified by the [=credential ID=]. They are also visible
+    to the [=client=] in the communication with the [=authenticator=].
+
+- The user's identities specific to each [=[RP]=], e.g. usernames, [=user handles=] and similar aliases.
+
+    These identities are obviously used by each [=[RP]=] to identify a user in their system. They are also visible to the
+    [=client=] in the communication with the [=authenticator=].
+
+- The user's biometric information, e.g., fingerprints or facial recognition data.
+
+    This is optionally used by the [=authenticator=] to perform [=user verification=]. It is not revealed to the [=[RP]=], but in
+    the case of [=platform authenticators=] it might be visible to the [=client=] depending on the implementation.
```

Comment:
Add the missing comma after "[=platform authenticators=].

Reply:
Fixed.